src/main/java/org/usfirst/frc4904/robot/commands/FlywheelSpinDown.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc4904.robot.commands;
+
+import org.usfirst.frc4904.robot.subsystems.Flywheel;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class FlywheelSpinDown extends CommandBase {
```

Comment:
should this also be a `motorVelocitySet`?

Reply:
No, because it should just set the speed of the motor 0 (we don't actually care if the motor has some excess speed, so long as it slowly approaches 0).